src/index.js

```diff
@@ -191,7 +204,10 @@ export default class RolloverTodosPlugin extends Plugin {
           file: file,
           oldContent: `${dailyNoteContent}`
         }
-        const todos_todayString = `\n${todos_today.join('\n')}`
+        todos_todayString = `\n`
+        todos_today.forEach(element => {
+          todosAddedString = todosAddedString + element.leading_whitespace + element.content + `\n`
+        });
```

Comment:
It seems to me that `todosAddedString` is not declared nor used.
Didn't you want to do something like:
```javascript
        todos_today.forEach(element => {
          todos_todayString += element.leading_whitespace + element.content + `\n`
        });
```